src/radical/pilot/states.py

```diff
@@ -41,13 +41,14 @@
         CANCELED               :  5}
 _pilot_state_inv   = {v: k for k, v in _pilot_state_values.iteritems()}
 _pilot_state_inv_full = dict()
-for s,v in _pilot_state_values.iteritems():
+for st,v in _pilot_state_values.iteritems():
     if v in _pilot_state_inv_full:
```

Comment:
why the change from s to st?

Reply:
The linter was complaining because `s` is used also as a parameter to a function.

Reply:
Why the hell is it complaining about that?
```
   list.append(s)
   list[len(list)] = s
```
would be equivalent, but it would not complain about the latter?  That does not make sense...  Lets not follow the linter blindly, please...

Reply:
The issue was this:
```
for s,v in _pilot_state_values.iteritems():
    if v in _pilot_state_inv_full:
        if not isinstance(_pilot_state_inv_full[v], list):
            _pilot_state_inv_full[v] = [_pilot_state_inv_full[v]]
        _pilot_state_inv_full[v].append(s)
    else:
        _pilot_state_inv_full[v] = s


def _pilot_state_value(s):
    return _pilot_state_values[s]
```
And this was the linter's message:
`W: 53,23: Redefining name 's' from outer scope (line 44) (redefined-outer-name)`

We can take it out if you think this is too much.

Reply:
yes, thank you.